DockeriseRmap/description.txt

```diff
@@ -0,0 +1,27 @@
+Run with:
+docker run --rm -v /workspaces/ExpansionHunter/results:/results:ro -v /workspaces/ExpansionHunter/results:/results2:rw -ti <Image ID>
```

Comment:
Could this be converted to a properly formatted markdown file please?

Reply:
Done